k8s.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: prebid-server
+  spec:
+    replicas: 4
+    template:
+      metadata:
+        labels:
+          app: prebid-server
+          type: microservice
+      spec:
+        containers:
+        - name: prebid-server
+          image: eu.gcr.io/spil-infra-registry/prebid-server:0000000000
+          ports:
+          - containerPort: 5000
+          resources:
+            requests:
+              cpu: 100m
+              memory: 600Mi
```

Comment:
should look into actual usage and probably lower memory (and perhaps increase cpu)